rc4me/run.py

```diff
@@ -1,39 +1,29 @@
 """TODO - Docstring."""
 
-import sys
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Dict
 import logging
 import click
 
-from rc4me.util import RcDirs, link_files
-from rc4me.prepare_repo import prepare_repo
+from rc4me.util import RcDirs, fetch_repo, link_files
 
 logging.basicConfig(level=logging.DEBUG)
 logger = logging.getLogger(__name__)
 
 
-@click.command()
+@click.group()
 # Allow calling function without an argument for --revert or --reset options
-@click.argument("repo", required=False, type=str)
 @click.option("--dest", type=click.Path())
 @click.option("--home", type=click.Path())
-@click.option("--revert", is_flag=True)
-@click.option("--reset", is_flag=True)
+@click.pass_context
 def cli(
-    repo: Optional[str] = None,
+    ctx: Dict[str, RcDirs],
     home: Optional[str] = None,
     dest: Optional[str] = None,
-    revert: bool = False,
-    reset: bool = False,
 ) -> None:
-    """Fetch dotfile repo and link rc files to destination."""
+    """Initialize rc4me home directory and call rc4me subcommands."""
```

Comment:
I don't love this docstring, but nothing great is coming to me. This should just hint that this is a master command:

<img width="444" alt="image" src="https://user-images.githubusercontent.com/12516153/105550620-7ca62a80-5cb6-11eb-8518-11ea52bf5e53.png">

Reply:
Maybe just a tagline here?

"""Management for run commands""""

Reply:
You can make the current doc string a commend? Honestly not totally sure. Curious what you think.